### PR TITLE
Add support for city, address, address2 fields

### PR DESCRIPTION
**Pull request by solocommand**
Add support for setting/displaying City, Address, and Address 2 fields.

Due to the number of fields, moved the address-specific fields to their own route/tab on the app user modal.

image
image

### CODE REVIEW

**Comment by solocommand** [comment]
[Fields on Identity X to set to required](https://trello.com/c/dM6HfndK/257-fields-on-identity-x-to-set-to-required)

File: packages/mongoose-plugins/src/locale-plugin.js [hunk 1]
```diff
@@ -55,6 +55,18 @@ module.exports = function localePlugin(schema, { localeService } = {}) {
         message: 'Invalid postal code {VALUE} for the provided country.',
       },
     },
+    city: {
+      type: String,
+      trim: true,
+    },
+    address: {
```
Comment (zarathustra323) [inline packages/mongoose-plugins/src/locale-plugin.js]: This should be `street`

File: packages/mongoose-plugins/src/locale-plugin.js [hunk 2]
```diff
@@ -55,6 +55,18 @@ module.exports = function localePlugin(schema, { localeService } = {}) {
...
+      type: String,
+      trim: true,
+    },
+    address2: {
```
Comment (zarathustra323) [inline packages/mongoose-plugins/src/locale-plugin.js]: This should be `addressExtra`

File: services/graphql/src/graphql/definitions/app-user.js [hunk 3]
```diff
@@ -73,6 +73,9 @@ type AppUser {
   region: LocaleRegion @projection(localField: "regionCode", needs: ["countryCode"])
   regionCode: String @projection
   postalCode: String @projection
+  city: String @projection
+  address: String @projection
```
Comment (zarathustra323) [inline services/graphql/src/graphql/definitions/app-user.js]: Adjust field names to match the new model fields  (and in the input below)

File: services/manage/app/templates/components/app-user/fields/region.hbs [hunk 4]
```diff
@@ -1,4 +1,4 @@
-<FormElements::Label @for="app-user.region">State / Region</FormElements::Label>
+<FormElements::Label @for="app-user.region">State / Province</FormElements::Label>
```
Comment (zarathustra323) [inline services/manage/app/templates/components/app-user/fields/region.hbs]: Leave as `State / Region`

File: services/manage/app/templates/components/app-user/modal-form/address-fields.hbs [hunk 5]
```diff
@@ -0,0 +1,36 @@
+<div class="row">
+  <div class="col-sm-6">
+    <div class="form-group">
+      <FormElements::Label @for="app-user.address">Address</FormElements::Label>
```
Comment (zarathustra323) [inline services/manage/app/templates/components/app-user/modal-form/address-fields.hbs]: Change label to `Street`

File: services/manage/app/templates/components/app-user/modal-form/address-fields.hbs [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+      <Input @id="app-user.address" @type="text" @class="form-control" @value={{this.model.address}} />
+    </div>
+  </div>
+  <div class="col-sm-6">
+    <div class="form-group">
+      <FormElements::Label @for="app-user.address2">Apartment, suite, etc</FormElements::Label>
```
Comment (zarathustra323) [inline services/manage/app/templates/components/app-user/modal-form/address-fields.hbs]: Change label to `Extra (Apt, Suite, etc.)`

**Review by zarathustra323 — CHANGES_REQUESTED** [review]

**Comment by solocommand** [comment]
@zarathustra323 Changes implemented:
image

**Review by zarathustra323 — APPROVED** [review]

**Pull request merged**